benefit/data/sheets/CaseEventToFields/CaseEventsToFields-writeAdjournmentNotice.json

```diff
@@ -0,0 +1,332 @@
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "whenShouldNextHearingBe",
+    "PageLabel": "When should the next hearing be?",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseNextHearingDateOrPeriod",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "adjournCaseNextHearingDateType = \"firstAvailableDateAfter\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "whenShouldNextHearingBe",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseNextHearingFirstAvailableDateAfterDate",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "adjournCaseNextHearingDateOrPeriod = \"provideDate\" AND adjournCaseNextHearingDateType = \"firstAvailableDateAfter\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 4,
+    "PageID": "whenShouldNextHearingBe",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseNextHearingFirstAvailableDateAfterPeriod",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "adjournCaseNextHearingDateOrPeriod = \"providePeriod\" AND adjournCaseNextHearingDateType = \"firstAvailableDateAfter\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 5,
+    "PageID": "whenShouldNextHearingBe",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    ",ShowSummaryChangeOption": "Y",
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseTime",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "adjournCaseNextHearingDateType = \"firstAvailableDate\" OR adjournCaseNextHearingDateType = \"firstAvailableDateAfter\" OR adjournCaseNextHearingDateType = \"dateToBeFixed\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 6,
+    "PageID": "whenShouldNextHearingBe",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseReasons",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 14,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "reasons",
+    "PageLabel": "Reasons for adjournment",
+    "PageShowCondition": "adjournCaseGenerateNotice = \"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CallBackURLMidEvent": "${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEventPreviewAdjournCase",
```

Comment:
can we follow the same naming convention as they've done for the other midEvent
${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEventAdjournCasePopulateVenueDropdown